releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,16 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.0.0 - xx xx, 2023
+
+### Breaking Changes
+* Schema (more information in [Azure App Configuration Kubernetes Provider reference](https://learn.microsoft.com/en-us/azure/azure-app-configuration/reference-kubernetes-provider?tabs=default)):

```

Comment:
Schema update (see \[Azure App Configuration Kubernetes Provider reference](https://learn.microsoft.com/azure/azure-app-configuration/reference-kubernetes-provider) for the complete schema):